### DISCUSSION TITLE
How to setup the default repository on GitHub Desktop?

### DISCUSSION THREAD

**Discussion by PaulStSmith** [community/community#40157] [New to GitHub]
I searched but could not find a post about this. In any case I want to apologize if I'm duplicating a thread or posting it in the wrong place.

Anyway, I use GitHub Desktop and I have a few cloned repos in my machine. However, I most often interact with only one of them, as the others are reference projects I use, and I'm tired of manually changing the repo to do so.

So I wanted to set my most frequent repo as the default when I open GitHub Desktop, but I could not find the right setting.

How is it done? Or is it even possible?

**Comment by melvincwng** [comment]
Hey @PaulStSmith, you might wish to check out the following GitHub Documentation links below to know more about the usage of GitHub Desktop:

1. https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/creating-your-first-repository-using-github-desktop
2. https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/adding-a-repository-from-your-local-computer-to-github-desktop

The first link gives an overview of the usage of GitHub Desktop while the second link might address the issue you're facing currently.

Hope this answer is useful and has helped you, take care  😊

**Reply by PaulStSmith** [reply]
The point is @melvincwng: creating a repository, or adding it to GitHub is not a problem. I have them.

My problem is when I open GitHub Desktop I want it to show my preferred repo not the first one in alphabetical order.

**Reply by melvincwng** [reply]
@PaulStSmith, try adding & opening your preferred repository on GitHub Desktop first, then open your IDE of choice and make some changes. Go back to GitHub Desktop, commit & push those changes.

After closing your GitHub Desktop, subsequent reopening of the software should show that preferred repo of yours. At least, on my end, this is how GitHub Desktop works.

Hope this is helpful, otherwise you might want to wait for another response.

**Comment by PaulStSmith** [comment]
I'll try, and let you know how it goes.

**Comment by jfullstackdev** [comment]
how hard is it to change repos ? it's just 2 clicks away, isn't it ? 

![changingrepos](https://user-images.githubusercontent.com/47092464/204117520-d40690e1-9ea6-477a-802a-426455d7fffb.png)

and the above post is correct, once you are in one repo, it will be the same repo when you open it

I'm guessing too, if you are switching repos from time to time, that's not a common situation for developers, even in custom software development, usually you will be assigned with just one project, so you don't really clone too many repos in a single time, as for my situation, for almost 4 months, I'm still on the same project and that project started last year and still in development, developers can be pulled out but it's not on everyday or weekly basis, so, again, you will not end up with too many local clones

**Comment by xoxocandykisses100** [comment]
New License needs to update. Old Generator and radio Elimination due to not
following direct orders from their Supervisors Initial Direct Orders to
Stand Down. only Exception is the Disability of 1 Loretta Lyons Villarreal.
He said terminate the insubordination of The Federal Government Branches
